### ISSUE TITLE
[SUGGESTION] Different Model and Different sprite/icon/in gui for items

### ISSUE THREAD

**Issue by MarkinhosMLP** [oraxen/oraxen#667]
### I have checked that...

- [X] ...such a feature does not exist already
- [X] ...such a feature request has not been submitted already

### Description

just like tridents, it is possible to make items to have a different texture/model in the gui and other in hand
but it needs vanilla shaders, just like the armors

it is actually the same json, but with different cubes/groups. it uses translucency to determine which one will be displayed in the gui and which one will be displayed in hand

you can see more about it searching for CorePerspectiveModels
![image](https://user-images.githubusercontent.com/47507632/215232871-12fd1764-f784-4c4b-b8dc-2fbc2f5d6d23.png)
![image](https://user-images.githubusercontent.com/47507632/215232887-2a252258-a374-4a87-84a7-d054ee08b5f8.png)
![image](https://user-images.githubusercontent.com/47507632/215232898-f36860f9-215b-4e41-a636-e9fc37e644fa.png)
![image](https://user-images.githubusercontent.com/47507632/215232907-1113a336-3a1a-4d2e-8db2-b04d559abe3f.png)
![image](https://user-images.githubusercontent.com/47507632/215232936-32f7922a-62dc-4436-aab6-ea2be27083a7.png)

### Reason

this would be really cool... since you can make armors in the vanilla way and oraxen converts it automatically into shaders version would also be great

as i said, it uses translucency so, in settings you could define something like

gui-item-alpha: 253
in-hand-item-apha: 254

**Comment by Boy0000** [comment]
the shader used here breaks on alot of systems due to outdated drivers
This can be done manually and is too broken to automate imo
The shader is also not foul proof and unlike armor, entirely breaks afaik when using optifine or iris

TLDR if u want this, manually add the shader and handle the files as needed
